Dockerfile

```diff
@@ -1,14 +1,12 @@
-FROM phusion/baseimage as amoc-build
+FROM phusion/baseimage:18.04-1.0.0 as amoc-build
```

Comment:
As in https://github.com/esl/amoc-arsenal-xmpp/pull/20, I'd actually change it to `bionic-1.0.0`. Checking it on github, they both actually point to the same commit hash, so they're equivalent, but the `bionic-` version is the one they have released as an "official" tag.

Reply:
I would rather leave the YY.MM version, I usually do not remember which "animal" correspond to which ubuntu release. but now it's pretty clear how old the version is.